distro/pom.xml

```diff
@@ -28,15 +28,15 @@
     <initializer.version>2.5.0</initializer.version>
     <webservices.rest.version>2.39.0</webservices.rest.version>
     <addresshierarchy.version>2.15.1</addresshierarchy.version>
-    <idgen.version>4.10.0-SNAPSHOT</idgen.version>
+    <idgen.version>4.10.0</idgen.version>
     <legacyui.version>1.15.0</legacyui.version>
     <metadatamapping.version>1.6.0</metadatamapping.version>
     <metadatasharing.version>1.9.0</metadatasharing.version>
-    <openconceptlab.version>2.1.0-SNAPSHOT</openconceptlab.version>
+    <openconceptlab.version>2.0.0</openconceptlab.version>
```

Comment:
@rbuisson, the 2.1.0 release was given a green light [on talk](https://talk.openmrs.org/t/o3-release-list-of-backend-modules/39494/3). Could we use 2.1.0 instead?